packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
```

Comment:
Shouldn't we move the `mini` and `sub` into a `size` prop?
Otherwise, you could end with something like `<ListItem mini sub />` (as raised here https://dev.to/selbekk/the-10-component-commandments-2a7f)
It also makes the API more readable

Reply:
👍